qiskit_nature/algorithms/pes_samplers/bopes_sampler.py

```diff
@@ -132,6 +173,14 @@ def sample(self, problem: BaseProblem, points: List[float]) -> BOPESSamplerResul
         if self._driver.molecule is None:
             raise QiskitNatureError("Driver MUST be configured with a Molecule.")
 
+        # force the initialization of the solver before starting the computation
+        if isinstance(self._gss, MinimumEigensolverFactory):
+            if self._qubit_converter is None:
+                raise QiskitNatureError("When using MinimumEigensolverFactory "
+                                        "you must specify a qubit converter")
+            prepare_problem(problem, self._qubit_converter)
+            self._solver = self._gss.get_solver(problem,self._qubit_converter)
```

Comment:
Initialize solver before the tricks on initial_point. We have to apply second quantisation on the problem to obtain the solver.